app/models/host/managed.rb

```diff
@@ -797,6 +797,10 @@ def global_status_label(options = {})
     HostStatus::Global.build(host_statuses, options).to_label
   end
 
+  def global_status_fulltext
+    host_statuses.select { |s| s.relevant? && !s.substatus? }.sort_by(&:type).map { |s| "#{_(s.name)}: #{_(s.to_label)}" }.join('. ')
```

Comment:
I know that what you have here is equivalent to the old page. But I think with very small changes, we can make it 100% better.

```suggestion
    host_statuses.select { |s| s.relevant? && !s.substatus? }.sort_by(&:type).map { |s| "#{_(s.name)}: #{_(s.to_label)}" }
```

If we send an array instead of a concatenated string, we can iterate over that array on the frontend. That way each individual status message can at least be on its own line in the popover.